source/fuzz/transformation_load.cpp

```diff
@@ -153,9 +153,8 @@ bool TransformationLoad::IsApplicable(
       return false;
     }
 
-    // The memory semantics constant value must be either
-    // SpvMemorySemanticsWorkgroupMemoryMask or
-    // SpvMemorySemanticsUniformMemoryMask.
+    // The memory semantics constant value must match the storage class of the
+    // pointer being loaded from.
     auto memory_semantics_const_value =
```

Comment:
I think it might prove necessary to declare this as having type SpvMemorySemanticsMask, and static_cast it to that type. I bet some of the bots will complain with an error about there being an implicit cast.

Reply:
Done.